packages/web/src/pages/sign-up-page/pages/CreatePasswordPage.tsx

```diff
@@ -38,14 +35,13 @@ type CreatePasswordValues = {
 }
 
 type CreatePasswordPageProps = {
-  params: { email: string }
   onPrevious: (state: SignUpState) => void
   onNext: (state: PickHandleState) => void
 }
 
 export const CreatePasswordPage = (props: CreatePasswordPageProps) => {
-  const { params, onNext } = props
```

Comment:
Will create some small some merge conflicts with Nikki's PR #6443 so I guess whoever merges last gets to deal with it 😜 
@nicoback2